docs/python_docs/python/tutorials/packages/np/cheat-sheet.md

```diff
@@ -90,7 +90,7 @@ npx.load('my_array')
 ```{.python .input}
 # Save a list of arrays
 b = np.array([4, 6, 8])
-npx.save('my_arrays', [a, b])  # FIXME, cannot be a tuple
+npx.savez('my_arrays', *[a, b])  # FIXME, cannot be a tuple
```

Comment:
Is this comment still relevant?

Reply:
Not relevant. Let me remove the comment.